environment/s3.tf

```diff
@@ -70,11 +106,7 @@ resource "aws_s3_bucket_public_access_block" "pa_uploads" {
   restrict_public_buckets = true
 }
 
-resource "aws_s3_bucket_policy" "pa_uploads" {
-  bucket = aws_s3_bucket_public_access_block.pa_uploads.bucket
-  policy = data.aws_iam_policy_document.pa_uploads.json
-}
-
+// BASE POLICY FOR PA_UPLOADS
```

Comment:
I find these comments throughout super useful - its sometimes hard to look at terraform and know what its doing without having the docs up 👍

Reply:
Yeah they're a bit ugly but I actually find them useful myself too so left them in.